gateway/appconfig/appconfig.go

```diff
@@ -89,19 +98,22 @@ func Load() error {
 		webappUsersManagement = "on"
 	}
 	runtimeConfig = Config{
-		apiURL:                fmt.Sprintf("%s://%s", apiRawURL.Scheme, apiRawURL.Host),
-		apiHostname:           apiRawURL.Hostname(),
-		apiScheme:             apiRawURL.Scheme,
-		apiHost:               apiRawURL.Host,
-		askAICredentials:      askAICred,
-		pgCred:                pgCred,
-		migrationPathFiles:    migrationPathFiles,
-		licenseSigningKey:     licensePrivKey,
-		licenseSignerOrgID:    allowedOrgID,
-		gcpDLPJsonCredentials: gcpJsonCred,
-		webhookAppKey:         os.Getenv("WEBHOOK_APPKEY"),
-		webappUsersManagement: webappUsersManagement,
-		isLoaded:              true,
+		apiURL:                  fmt.Sprintf("%s://%s", apiRawURL.Scheme, apiRawURL.Host),
+		apiHostname:             apiRawURL.Hostname(),
+		apiScheme:               apiRawURL.Scheme,
+		apiHost:                 apiRawURL.Host,
+		askAICredentials:        askAICred,
+		pgCred:                  pgCred,
+		migrationPathFiles:      migrationPathFiles,
+		licenseSigningKey:       licensePrivKey,
+		licenseSignerOrgID:      allowedOrgID,
+		gcpDLPJsonCredentials:   gcpJsonCred,
+		dlpProvider:             dlpProvider,
+		msPresidioAnalyzerURL:   msPresidioAnalyzerURL,
+		msPresidioAnonymizerURL: msPresidioAnonymizerURL,
```

Comment:
You can pass the `os.Getenv("<ENV>")` directly, in case it doesn't exists the default value will be a empty string. There's no need to create a function to parse those values.